crucible-llvm/src/Lang/Crucible/LLVM/Intrinsics/Common.hs

```diff
@@ -95,13 +95,13 @@ data LLVMOverride p sym args ret =
          bak ->
          Ctx.Assignment (RegEntry sym) args ->
          forall rtp args' ret'.
-         OverrideSim p sym LLVM rtp args' ret' (RegValue sym ret)
+         OverrideSim p sym ext rtp args' ret' (RegValue sym ret)
     -- ^ The implementation of the intrinsic in the simulator monad
     -- (@OverrideSim@).
   }
 
 data SomeLLVMOverride p sym =
-  forall args ret. SomeLLVMOverride (LLVMOverride p sym args ret)
+  forall args ret. SomeLLVMOverride (LLVMOverride p sym LLVM args ret)
```

Comment:
Would it make more sense for `SomeLLVMOverride` to be parameterized by `ext`, just like `LLVMOverride`? We _could_ force it to be `LLVM`, but I worry that someone will come along later and want the same generalizations that this PR is applying to `LLVMOverride`.